src/asciidoctor-core-api.js

```diff
@@ -323,7 +323,7 @@ Block.$$proto.getSource = function () {
  * @returns {Array} - returns the String {Array} of source lines for this block.
  */
 Block.$$proto.getSourceLines = function () {
-  return this.$lines();
+  return this.lines;
```

Comment:
I think both are working if the getter/setter is not overridden because we do have a test case on this function: https://github.com/asciidoctor/asciidoctor.js/blob/eac4c2e30b90f3a16661ebb4f87c2502e2c9b357/spec/node/asciidoctor.spec.js#L153

But it's better to use `this.lines` because `lines` is defined as an attribute accessor in Asciidoctor core.

Reply:
The biggest problem was the API was doing different things for different methods. I think we should use the shortest path to properties for best performance, which is the form `this.lines` as you mentioned.